### PR TITLE
Chapter 11 : Account Activation

### PR DESCRIPTION
**Pull request by DaudKhadijah01**

### CODE REVIEW

**Comment by DaudKhadijah01** [comment]
ready

File: app/assets/stylesheets/account_activations.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/account_activations.scss]: delete this file

File: app/controllers/account_activations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by email: params[:email]
+    if user && !user.activated && user.authenticated?(:activation, params[:id])
+      user.update_attribute :activated, true
+      user.update_attribute :activated_at, Time.zone.now
```
Comment (ngocvt-0484) [inline app/controllers/account_activations_controller.rb]:
```suggestion
      user.update_attributes activated_at: Time.zone.now, activated: true
```

File: app/helpers/account_activations_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (ngocvt-0484) [inline app/helpers/account_activations_helper.rb]: delete this

File: test/controllers/account_activations_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class AccountActivationsControllerTest < ActionDispatch::IntegrationTest
```
Comment (ngocvt-0484) [inline test/controllers/account_activations_controller_test.rb]: delete all files on test folder
it not use

**Comment by DaudKhadijah01** [comment]
ready

**Comment by DaudKhadijah01** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**